### PR TITLE
DOC: Link to CONTRIBUTING.md in README.md

### PR DESCRIPTION
**Pull request by mp3403**
Added link to CONTRIBUTING.md in the "Contributions" section and added three common questions and answers in the Q&A section.

### CODE REVIEW

**Comment by stefan6419846** [comment]
Where do these FAQ come from? If ever, shouldn't these rather be part of the official docs instead of the README?

Additionally, the `extractText` method is deprecated in favor of `extract_text`: https://pypdf.readthedocs.io/en/latest/modules/PageObject.html?pypdf._page.PageObject.extractText#pypdf._page.PageObject.extractText

File: README.md [hunk 1]
```diff
@@ -71,8 +72,22 @@ want to make their live easier to experts who developed software before PDF
...
+The following is some of the most commonly asked questions:
 
+**I get the following error message when I use the pypdf module: <code>ImportError: No module named 'pyPDF'</code>**
+
+To fix this issue, be sure that you are using the correct name when importing the module. The name of the module is <code>pypdf</code>.
+Also, be sure you are using the correct version of pypdf. pypdf 3.1.0 and above are recommended. Use <code>pypdf>=3.1.0</code> if the issue persists.
+
+**How can I use pypdf to merge pdf files?**
+
+Use the <code>PdfMerger</code> class. The <code>append</code> method is great for simply concatenating files, but <code>merge</code> can be used
```
Comment (pubpub-zz) [inline README.md:85]: PdfMerger is obsolete. PdfWriter has to be used instead

File: README.md [hunk 2]
```diff
@@ -71,8 +72,22 @@ want to make their live easier to experts who developed software before PDF
...
+for more precise control.
+
+**How can I extract text from a pdf?**
+
+There are several ways of extracting text from a pdf file using pypdf. The <code>extractText</code> method is likely the simplest way of doing this.
```
Comment (pubpub-zz) [inline README.md:90]: link to the proper documentation page should be added

**Review by pubpub-zz — CHANGES_REQUESTED** [review]

**Comment by MartinThoma** [comment]
The link to CONTRIBUTING is fine. Please remove the rest.

File: README.md [hunk 3]
```diff
@@ -71,8 +72,22 @@ want to make their live easier to experts who developed software before PDF
 existed. You can contribute to the pypdf community by answering questions
 on [StackOverflow](https://stackoverflow.com/questions/tagged/pypdf),
 helping in [discussions](https://github.com/py-pdf/pypdf/discussions),
-and asking users who report issues for [MCVE](https://stackoverflow.com/help/minimal-reproducible-example)'s (Code + example PDF!).
+and asking users who report issues for [MCVE](https://stackoverflow.com/help/minimal-reproducible-example)'s (Code + example PDF!).  
+The following is some of the most commonly asked questions:
```
Comment (MartinThoma) [inline README.md:76]:
```suggestion
and asking users who report issues for [MCVE](https://stackoverflow.com/help/minimal-reproducible-example)'s (Code + example PDF!).
```

File: README.md (hunk 2 above)
Comment (MartinThoma) [inline README.md:90]:
```suggestion
```

**Comment by pubpub-zz** [comment]
@MartinThoma  Have you consiredered my comments ?

**Comment by MartinThoma** [comment]
@pubpub-zz I have removed those parts from the PR as they are already covered in a cleaner way in the very top of the PR.

**Comment by pubpub-zz** [comment]
all good, I was just questionning based on the replies to my comments.

**Comment by MartinThoma** [comment]
I do agree to the comments :+1:  But I don't think we should add complete paragraphs

**Pull request merged**